.travis.yml

```diff
@@ -30,10 +30,8 @@ before_install:
 
 # Install packages
 install:
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION --file requirements.txt
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION jupyter
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION pyflakes
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION pep8
+  - conda install --yes python=$TRAVIS_PYTHON_VERSION scipy matplotlib h5py jupyter
```

Comment:
Yeah, the aim of the above line was to create a test of the installation which is **as close as possible** to what the user types: see the other comment below. (except that we do not use `pip install openPMD-viewer` but `python setup.py install` to install the latest version of the code.)

Also, even with the above commands, `requirements.txt` will still be used (since setup.py is reading it when doing `python setup.py install`). The only reason why I was adding `conda install --file requirements.txt` is so that the requirements get installed with `conda` instead of `pip`. (again, see remark below)